src/embind/emval.js

```diff
@@ -126,6 +126,15 @@ var LibraryEmVal = {
     return Emval.toHandle([]);
   },
 
+  _emval_new_array_from_memory_view__deps: ['$Emval'],
```

Comment:
Can you add a `__sig` there.  I think it should be `ip`.

Reply:
What are `__sig`s doing?

Reply:
They have two purposes that I know of:

1. When adding a JS function to a wasm table it has to have known/fixed type.  This happens mostly in dynamic linking when a module take the address and of function that isn't already in the main module's table.
2. With -sMEMORY64/wasm64 we use `p` in the signature to automatically convert pointer/size_t values (i64) to JS numbers (i53) and the wasm boundary.  This avoids having to re-write JS code that expects pointers such that is can handle both Number and BigInt with the downside that limit pointers and size_t things to 53 bits.

Reply:
@sbc100 Why not `ii` or `pp`?  what's the difference?

Reply:
`i` is not 32-bit integers.   `p` is for pointer-sized things that are 32-bit on wasm32 and 64-bit on wasm64.  For example `size_t`, `long` and any pointer type.  Emscripten generates wrapper functions for any impors/exports that contains `p` in their signature.

Reply:
@sbc100 seems `ip` failed the `test-wasm64` and so I change it to `pp`. 
I just don't know how to run the failed test `test-wasm64` locally to verify the change.